modules/charon-core/src/main/java/org/wso2/charon3/core/attributes/DefaultAttributeFactory.java

```diff
@@ -28,15 +29,17 @@
  */
 public class DefaultAttributeFactory {
 
+    private static final Logger log = LoggerFactory.getLogger(DefaultAttributeFactory.class);
+
     /*
      * Returns the defined type of attribute with the user defined value
      * included and necessary attribute characteristics set
      * @param attributeSchema - Attribute schema
      * @param attribute - attribute
      * @return Attribute
      */
-    public static Attribute createAttribute(AttributeSchema attributeSchema,
-                                            AbstractAttribute attribute) throws CharonException, BadRequestException {
+    public static Attribute createAttribute(AttributeSchema attributeSchema, AbstractAttribute attribute)
```

Comment:
We are not recommending to change public methods as we need to preserve the backward compatibility as these public methods can be used from different other places.